### DISCUSSION TITLE
Requesting a antibuild permission node functionality/feature

### DISCUSSION THREAD

**Discussion by Onomastically** [EssentialsX/Essentials#4577] [Ideas and Feature Suggestions] [answered]
Can we possibly get a way to use EssentialsX Antibuild to have an 'exempt-a-specific-item-from-the-antibuild-black-list-function?

Example: essentials.protect.exemptplacement.<ITEM>

We are only able to exempt for the entire placement/breaking/using of items in the list, not specific ones.

**Comment by mdcfe** [comment, marked as answer]
AntiBuild already has per-material permissions, so this is redundant. If you want to restrict items/blocks for specific players, you should negate the permission for that specific item/block (use `/ess debug` to show the permissions needed) instead of using the config blacklists.

**Reply by Onomastically** [reply]
I hate to say it, but I just double checked, and removing 'oak_sign' from the block blacklist on AntiBuild and then adding the permission node 'essentials.build.place.oak_sign' and then using LuckPerms editor to set the node to false did not actually prevent the group I set the permission to from placing oak wood signs.
